docs/content/tools/acquia-cli.md

```diff
@@ -22,8 +22,8 @@ generate your Acquia Cloud API Token.
 Add your secret and key with
 
 ```bash
-fin config set --global SECRET_ACQUIACLI_SECRET="lkjd*sdfl2//23498fwernoiuDHljw3897fsk"
-fin config set --global SECRET_ACQUIACLI_KEY="rSxVZN35bo4jTuncGS+CiKdmhxLPL0BaPuyOv"
+fin config set --global SECRET_ACQUIA_CLI_SECRET="lkjd*sdfl2//23498fwernoiuDHljw3897fsk"
+fin config set --global SECRET_ACQUIA_CLI_KEY="rSxVZN35bo4jTuncGS+CiKdmhxLPL0BaPuyOv"
```

Comment:
These probably should not be changed because `SECRET_ACQUIACLI_SECRET` and `SECRET_ACQUIACLI_KEY` are what are passed in automatically to the CLI and many users may already be using the existing variables. There is no functional reason to change them.